packages/@okta/vuepress-site/docs/reference/api/event-hooks/index.md

```diff
@@ -628,6 +637,59 @@ To use Basic Auth, set `type` to `HEADER`, `key` to `Authorization`, and `value`
 |----------|------------------------------------------------------------------------------|----------|----------|----------|
 | type     | The events object type. Currently the only supported type is `EVENT_TYPE`.   | String   | TRUE     | FALSE    |
 | items    | The [event types](#supported-events-for-subscription) to subscribe to       | Array of String  | TRUE     | FALSE    |
+| filter    | The filter defined on a specific event type    | [filter object](#filter-object)  | FALSE     | FALSE    |
+
+##### Example of events object
+
+```json
+
+"events": {
+            "type": "EVENT_TYPE",
+            "items": [
+                "user.session.end",
+                "user.session.start"
+            ],
+            "filter": {
+                "type": "EXPRESSION_LANGUAGE",
+                "eventFilterMap": [
+                    {
+                        "event": "user.session.end",
+                        "condition": {
+                            "version": null,
+                            "expression": "event.eventType eq 'Admin' && event.eventType ne 'Bob'"
+                        }
+                    }
+                ]
+            }
+        }
+```
+
+#### Filter object
+
+| Property | Description                                                                  | DataType | Required | ReadOnly |
+|----------|------------------------------------------------------------------------------|----------|----------|----------|
+| type     | The type of filter. Okta only supports `EXPRESSION_LANGUAGE`   | String   | TRUE     | TRUE   |
+| eventFilterMap    | The object that maps the filter to the event type.      | Array of [eventFilterMap objects](#event-filter-map-object)  | FALSE    | FALSE    |
+
+#### Event filter map object
+
+<ApiLifecycle access="ea" />
+
+<EventHookEANote/>
+
+Explore the event hooks API with filters: [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/15901964-61517094-f0ed-45e9-982a-f7a6a2db1bab?action=collection%2Ffork&collection-url=entityId%3D15901964-61517094-f0ed-45e9-982a-f7a6a2db1bab%26entityType%3Dcollection%26workspaceId%3D9f1d6c8f-d027-4107-a5c5-20d963c2c9d8)
+
+| Property | Description                                                                  | DataType | Required | ReadOnly |
+|----------|------------------------------------------------------------------------------|----------|----------|----------|
+| event    | The filtered event type   | String   | TRUE     | FALSE   |
+| condition    | The object that defines the filter using Okta Expression Language      | [condition object](#condition-object)  | TRUE    | FALSE    |
```

Comment:
Would this be simpler as just something like "The object that defines the filter"?

The Condition object mentions OEL already.